### PR TITLE
Revisit the bundled block patterns

### PR DESCRIPTION
**Pull request by MaggieCabrera**
This is a follow-up on https://github.com/WordPress/gutenberg/issues/28921 since I discussed with Kjell that this would be better living in core with the rest of the patterns. Follow the discussion on the linked Gutenberg issue for all the details on these patterns. Please Kjell let me know if I should change the wording on the descriptions and titles of these patterns, as well as the categories if the patterns should be organized in a different way.

In order to ensure these patterns will work correctly on any theme, I tested them using [Empty Theme](https://github.com/WordPress/theme-experiments/tree/master/emptytheme), which adds no block CSS and no custom colors to the editor.

Trac ticket: https://core.trac.wordpress.org/ticket/52846

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by youknowriad** [comment]
Is this patch up to date with what has been merged in Gutenberg? 

Can we get the phpcs issues fixed?

**Comment by MaggieCabrera** [comment]
> Is this patch up to date with what has been merged in Gutenberg?
> 
> Can we get the phpcs issues fixed?

It is not! I will get to it tomorrow and be sure that the format is correct

**Comment by MaggieCabrera** [comment]
@youknowriad I just updated this with the latest from trunk in Gutenberg. Composer format didn't bring any errors for me.

File: src/wp-includes/block-patterns/heading.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Heading.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Heading', 'Block pattern title', 'default' ),
```
Comment (youknowriad) [inline src/wp-includes/block-patterns/heading.php:9]: I think these 'default' domain arguments should be dropped since we're already on Core.

Comment (youknowriad) [reply src/wp-includes/block-patterns/heading.php:9]: Maybe it's fine though. @swissspidy @gziolo do you know what is the guideline for Core?

Comment (gziolo) [reply src/wp-includes/block-patterns/heading.php:9]: It defaults to `default` so it can be dropped and in practice it usually is.

Comment (MaggieCabrera) [reply src/wp-includes/block-patterns/heading.php]: Thanks for the heads up, I just removed all of them 👍

**Comment by ryelle** [comment]
Committed in [r50794](https://core.trac.wordpress.org/changeset/50794).

**Pull request closed without merging**